app.js

```diff
@@ -49,6 +49,11 @@ app.use(
         }
     })
 );
+app.use(
```

Comment:
Can the helmet constructor/factory accept these arguments? Would allow us to remove some middleware calls